src/auth/auth.action.js

```diff
@@ -157,6 +159,34 @@ export const getUserEvents = user => {
   };
 };
 
+export const searchUserOpenPullRequests = () => {
```

Comment:
I suggest to rename as `getOpenPullRequest`, which is simpler and more similar with other functions.